Odata-docs/webapi-8/fundamentals/actions-functions.md

```diff
@@ -0,0 +1,423 @@
+---
+title: "Actions and functions"
+description: "This documentation describes how to add actions and functions."
+
+author: KenitoInc
+ms.author: kemunga
+
+ms.date: 9/15/2022
+---
+# Actions and functions
+**Applies To**:[!INCLUDE[appliesto-webapi](../../includes/appliesto-webapi-v8.md)]
+
+In OData, actions and functions are a way to add server-side behavior that are not easily defined as CRUD operations on entities. This documentation shows how to add actions and functions to an OData v4 endpoint, using ASP.NET Core OData v8.x.
```

Comment:
I think there was a request to explain what CRUD means, e.g., `CRUD operations (Create-Read-Update-Delete)`.